tests/cms/views/organizations/test_organization_form.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from django.test.client import Client
+    from pytest_django.fixtures import SettingsWrapper
+
+import pytest
+from django.test.client import Client
+from django.urls import reverse
+
+from integreat_cms.cms.models import MediaFile, Organization, Page, POI, Region
+from tests.conftest import ANONYMOUS, MANAGEMENT, STAFF_ROLES
+
+
+@pytest.mark.django_db
+def test_poi_form_shows_no_contents(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    settings: SettingsWrapper,
+) -> None:
+    """
+    Test that no contents are shown in the organization form if it does not have any content assinged
+    """
+    client, role = login_role_user
+
+    # Test for english messages
+    settings.LANGUAGE_CODE = "en"
+
+    new_organization = Organization.objects.create(
+        name="New Organization",
+        slug="new-organization",
+        icon=MediaFile.objects.filter(id=1).first(),
+        region=Region.objects.filter(id=1).first(),
+        website="https://integreat-app.de/",
+    )
+
+    edit_organization = reverse(
+        "edit_organization",
+        kwargs={
+            "slug": new_organization.slug,
+            "region_slug": new_organization.region.slug,
+        },
+    )
+    response = client.get(edit_organization)
+
+    if role in STAFF_ROLES + [MANAGEMENT]:
+        assert (
+            "This organization currently has no maintained pages."
+            in response.content.decode("utf-8")
+        )
+        assert (
+            "This organization currently has no maintained locations."
+            in response.content.decode("utf-8")
+        )
+
+    elif role == ANONYMOUS:
+        assert response.status_code == 302
+        assert (
+            response.headers.get("location")
+            == f"{settings.LOGIN_URL}?next={edit_organization}"
+        )
+    else:
+        assert response.status_code == 403
+
+
+@pytest.mark.django_db
+def test_poi_form_shows_associated_contents(
```

Comment:
```suggestion
def test_organization_form_shows_associated_contents(
```
I'm not entirely understanding this test yet. Am I understanding this correctly that we're testing the organization form, not the poi form?

Reply:
Yes, to test is the organization form. Sorry.